tests/negative/265/M.juvix

```diff
@@ -0,0 +1,16 @@
+module M;
+
+inductive Bool {
+  false : Bool;
+  true : Bool;
+};
+
+inductive Pair (A : Type) (B : Type) {
+  mkPair : A → B → Pair A B;
+};
+
+f : _ → _;
+f (mkPair false true) ≔ true;
+f true ≔ false;
```

Comment:
One little thing. If I test this file, the error is fine. One gets as expected:
```
The constructor true has type:
  Bool
but is expected to have type:
  Pair Bool Bool
```
However, if the function changes a bit to be:

```agda
f : _ → _;
f true ≔ false;
f (mkPair false true) ≔ true;
```

We get an error message that is not ok. It should be "... has type: Pair _ _" or something. Tbh, Idk rn how to make it simpler.

```
The constructor mkPair has type:
  Pair
but is expected to have type:
  Bool
```

Reply:
indeed. I was aware of this an I thought it was easy enough to understand, but on a second thought, it looks like a bug.
Maybe we should use a different error message when the arguments of the inductive type are irrelevant?
something like this:
```
The constructor mkPair belongs to the inductive type:
  Pair
but is expected to belong to the inductive type:
  Bool
```

Reply:
Exactly. Let's use this last suggestion. I'm fine with that for the moment. ;)

Reply:
Implemented in bd363933f7f5b860d729f062a106992e525b0896